File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Kiểm tra số nguyên tố/SNT.cs
using System;

namespace KiemTraSoNguyenTo
{
    public class Program
    {
        static void Main()
        {
            int a = int.Parse(Console.ReadLine());
            string check = "true";
            if (a < 2)
            {
                check = "false";
            }
            else
            {
                for (int i = 2; i <= Math.Sqrt(a); i++)
                {
                    if (a % i == 0)
                    {
                        check = "false";
                        break;
                    }
                }
            }
            Console.Write(check);
        }
    }
}

File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Kiểm tra số nguyên tố/SNT.cpp
#include<iostream>
#include<math.h>
using namespace std;
int main(){
    int n;
    int k ;
    cin >> n;
    for(int i =2;i<=sqrt((float)n);i++){
        if(n%i==0)
        k =1;
    }
    if(k ==1 or n<=1) cout << "false";
    else if(k!=1)  cout << "true";
    return 0;
}